mackerel-plugin-aws-rekognition/lib/aws-rekognition.go

```diff
@@ -0,0 +1,167 @@
+package mpawswaf
```

Comment:
Please change this package name following  [Travis failure report](https://travis-ci.org/mackerelio/mackerel-agent-plugins/builds/217702224#L1179). `mpawswaf` is used by plugin-aws-waf.